imgui-binding/build.gradle

```diff
@@ -34,3 +34,19 @@ jar {
         attributes  'Automatic-Module-Name': 'imgui.binding'
     }
 }
+
+tasks.register('makePropertyResource') {
+    def directory = file "${buildDir}/resources/main/imgui/"
```

Comment:
Can you explain how this works? I didn't see such approach, so I'd like to know details. Is this some sort of implicit extending of default gradle task.

Reply:
Calling tasks.register creates the `makePropertyResource` task. It likes to make a custom function and call it from the main class. This task creates `imgui/imgui-java.properties` which contains the version of imgui-java. The task is called only when you run `./gradlew makeProertyResource`, so it is made to be called from processResource task using dependsOn. It seems that processResource copies resources from `build/resources/main`, so we don't need to add the path to the main resource sets.